book/src/pong-tutorial/pong-tutorial-01.md

```diff
@@ -18,64 +23,60 @@ git = "https://github.com/amethyst/amethyst.git"
 features = ["vulkan"]
 ```
 
-In the `src` directory there's a `main.rs` file. Delete everything in that file,
-then add these imports:
+Alternatively, if you are developing on macOS, you might want to use `metal` rendering backend instead of `vulkan`. In this case you should change `features` entry in amethyst dependency.
```

Comment:
```suggestion
Alternatively, if you are developing on macOS, you might want to use the `metal` rendering backend instead of `vulkan`. In this case you should change the `features` entry in the `amethyst` dependency table.
```